File: Loopcom works/fix_bundle_edit.py
import os

p = os.path.join('app', 'dashboard', 'items', 'bundles', '[id]', 'edit', 'page.tsx')
with open(p, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# Find and replace lines 350-359 (0-indexed: 349-358)
new_lines = []
skip_until = -1
for i, line in enumerate(lines):
    if skip_until > 0 and i < skip_until:
        continue
    if '<Link href={`/dashboard/items/${bundleId' in line:
        # Replace lines 350-359 with simple back button
        new_lines.append('        <Button variant="ghost" size="sm" onClick={() => router.back()}>\n')
        new_lines.append('          <ArrowLeft className="mr-2 h-4 w-4" />\n')
        new_lines.append('          Back\n')
        new_lines.append('        </Button>\n')
        # Skip until after </Link>
        for j in range(i+1, len(lines)):
            if '</Link>' in lines[j]:
                skip_until = j + 1
                break
        continue
    new_lines.append(line)

with open(p, 'w', encoding='utf-8') as f:
    f.writelines(new_lines)

print(f'Fixed {p}')

File: Loopcom works/fix_bundle_edit.js
const fs = require('fs');
const p = 'app/dashboard/items/bundles/[id]/edit/page.tsx';
let content = fs.readFileSync(p, 'utf8');
const lines = content.split('\n');
const newLines = [];
let skip = false;
for (let i = 0; i < lines.length; i++) {
  if (skip) {
    if (lines[i].includes('</Link>')) { skip = false; }
    continue;
  }
  if (lines[i].includes('<Link href={') && lines[i].includes('bundleId')) {
    newLines.push('        <Button variant="ghost" size="sm" onClick={() => router.back()}>');
    newLines.push('          <ArrowLeft className="mr-2 h-4 w-4" />');
    newLines.push('          Back');
    newLines.push('        </Button>');
    skip = true;
    continue;
  }
  newLines.push(lines[i]);
}
fs.writeFileSync(p, newLines.join('\n'));
console.log('Done - fixed ' + p);
